src/components/TrocadilhoList/TrocadilhoList.test.js

```diff
@@ -0,0 +1,42 @@
+import { render, screen } from "@testing-library/react";
+import TrocadilhoList from "./TrocadilhoList";
+
+describe("<TrocadilhoList/>", () => {
+  const trocadilhosAvailable = [
+    {
+      date: "13/02/2023",
+      votes: 10,
+      dev: "cicero.medeirios",
+      context:
+        "Lorem ipsum dolor,sit alit. Fugiat animi quaerat assumenda architecto possimus qui itaque reprehenderit tempore provident debitis? Dicta blanditiis mollitia reiciendis saepe. Alias, facere? Excepturi, officia obcaecati.",
+      message: "Lorem ipsus",
+    },
+    {
+      date: "13/02/2023",
+      votes: 10,
+      dev: "cicero.medeirios",
+      context:
+        "Lorem ipsum dolor,sit alit. Fugiat animi quaerat assumenda architecto possimus qui itaque reprehenderit tempore provident debitis? Dicta blanditiis mollitia reiciendis saepe. Alias, facere? Excepturi, officia obcaecati.",
+      message: "Lorem ipsus",
+    },
+  ];
+  const trocadilhosEmpty = [];
+  it("A list of available trocadilhos should be displayed on the page", () => {
+    render(<TrocadilhoList trocadilhos={trocadilhosAvailable} />);
+
+    screen.logTestingPlaygroundURL();
+    const trocadilhosTable = screen.getByRole("table");
+
+    expect(trocadilhosTable).toBeInTheDocument();
```

Comment:
You should also have a expect for the table size (how many trocadilhos you have in one list).